pkg/asset/machines/worker.go

```diff
@@ -171,6 +175,17 @@ func defaultPowerVSMachinePoolPlatform() powervstypes.MachinePool {
 	}
 }
 
+func defaultNutanixMachinePoolPlatform() nutanixtypes.MachinePool {
+	return nutanixtypes.MachinePool{
+		NumCPUs:           2,
+		NumCoresPerSocket: 2,
+		MemoryMiB:         8192,
+		OSDisk: nutanixtypes.OSDisk{
+			DiskSizeMiB: 120 * 1024,
```

Comment:
Other platforms derive their volume size from `decimalRootVolumeSize`:
`DiskSizeMiB: decimalRootVolumeSize * 1024`

Reply:
Oh, nice. Will fix that.